messaging/messaging_batch.go

```diff
@@ -122,6 +249,8 @@ func (c *fcmClient) SendAllDryRun(ctx context.Context, messages []*Message) (*Ba
 // responses list obtained from the return value corresponds to the order of the input tokens. An
 // error from SendMulticast indicates a total failure -- i.e. the message could not be sent to any
 // of the recipients. Partial failures are indicated by a `BatchResponse` return value.
+//
+// Deprecated. Use SendEachForMulticast instead.
```

Comment:
I think this should be `Deprecated: Use SendEachForMulticast instead.`
See: https://go.dev/blog/godoc

Reply:
Good catch! The current usage of `Deprecated` in the SDK is actually incorrect. There is a fix in #531